packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 4.1.0 - Oct 17, 2022
+
+New features:
+
+- New scenario-based Teams app template `Workflow bot` that enables you to create workflow style Teams application to facilitate users progress through multiple Adaptive Cards via buttons and inputs. Checkout our [documentation](https://aka.ms/teamsfx-card-action-response) for more details.
+- Debugging process is now transparently defined in `.vscode/tasks.json` and accepts parameters and arguments to enable customization and provide more granular control over debugging processes. Checkout our [documentation](https://aka.ms/teamsfx-debug-tasks) for more details. If you are running an existing project created by older Teams Toolkit versions, checkout this guide for [how to upgrade](https://aka.ms/teamsfx-debug-upgrade-new-tasks).
+- Simplified single sign-on for Message Extension and Command bot via `Add sso` command and TeamsFx SDK that helps you leverage logged-in Teams user token when accessing Microsoft Graph from your Teams application. Checkout our [documentation](https://aka.ms/teamsfx-add-sso) for more details.
```

Comment:
Use the new **Add SSO** command for a simple way to [configure single sign-on for Message extension and Command bot projects](https://aka.ms/teamsfx-add-sso).